content/chapters/compute/lab/support/wait-for-me/wait_for_me_threads.c

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BSD-3-Clause
+#include <stdio.h>
+
+#define ARR_LEN 400
+
+static inline void negate_array(int *arr, size_t len)
+{
+	for (int i = len - 1; i != -1; --i)
+		arr[i] = -arr[i];
+}
+
+int main(void)
+{
+	int arr[ARR_LEN];
+
+	for (int i = 0; i < ARR_LEN; ++i)
+		arr[i] = i + 1;
+
+	/* TODO: Call the `negate_array()` function in a separate thread.` */
```

Comment:
```suggestion
	/* TODO: Call the `negate_array()` function in a separate thread. */
```
Sorry, I've just seen this now :clown_face:.